Kitodo/src/main/java/org/kitodo/production/forms/createprocess/ProcessTextMetadata.java

```diff
@@ -43,7 +43,7 @@ public ProcessTextMetadata(ProcessTextMetadata template) {
     }
 
     private String addLeadingZeros(String value) {
-        if (Objects.equals(super.settings.getInputType(), InputType.INTEGER)) {
+        if (Objects.nonNull(super.settings) && InputType.INTEGER.equals(super.settings.getInputType())) {
```

Comment:
I would like better:

    Objects.nonNull(super.settings) && Objects.equals(InputType.INTEGER, super.settings.getInputType())
Not a requirement, but I think it is better readable. And, Objects.equals does `null`-checks.

Reply:
In this particular case I disagree. The additional `Objects.equals` is not required and unnecessarily complicates this check, since `InputType.INTEGER` is not null and therefor we do not need a null check here (calling the `equals` method on contants or enums when comparing with variables is an encouraged way for avoiding null pointer exceptions). So if you do not mind I would leave this part unchanged.